README.md

```diff
@@ -42,13 +42,39 @@ npm run build && npm run build:docs
 <!-- prettier-ignore-start -->
 <!-- toc -->
 * [Table of Contents](#table-of-contents)
+* [Quick Start](#quick-start)
 * [CLI Configuration](#cli-configuration)
 * [GitHub Actions Configuration](#github-actions-configuration)
-* [Usage](#usage)
 * [Command Topics](#command-topics)
 <!-- tocstop -->
 <!-- prettier-ignore-end -->
 
+# Quick Start
+
+Install the CLI ([see here for more setup options](#setup)):
+
+```sh
+npm install -g rdme
+```
+
+Validate an OpenAPI file in your working directory or any subdirectories ([see here for all command topics](#command-topics)):
+
+```sh
+rdme openapi validate
+```
+
+Every command has a help page, which you can access in [our docs](./documentation/commands) or via the CLI:
+
+```sh
+rdme openapi validate --help
```

Comment:
v9 has space separated commands? i thought we were doing that starting in v10?

Reply:
oh nope sorry i did that in v9 just so it'd be easy to maintain our v9 and v10 documentation in parallel — any qualms with that?

Reply:
nope was just confused